readable_text.js

```diff
@@ -10,13 +13,18 @@ javascript: (() => {
   /* process all children of a Node*/
   const HalfBold = (parentElement) => {
     /* iterating through all children of the parent*/
+
     for (var i = 0; parentElement.childNodes[i] != undefined; i++) {
       /* if the child is a text element*/
       if (parentElement.childNodes[i].nodeName == "#text" &&
         parentElement.childNodes[i].textContent.trim().length != 0
       ) {
-        var recentNode = parentElement.childNodes[i];
+
+        var recentNode = parentElement.childNodes[i]; // We cannot modify the text node but we will modify it's parent element
```

Comment:
```suggestion
        var recentNode = parentElement.childNodes[i]; /* We cannot modify the text node but we will modify it's parent element */
```
Open ended comments break the bookmarklet, even though they work for the extension.